src/pages/TypingTest/container.js

```diff
@@ -0,0 +1,10 @@
+import { connect } from 'react-redux';
+
+import Main from './index';
+
+const mapStateToProps = (state) => ({
+  preferences: state.userPreferences.preferences,
+  preferencesLoading: state.userPreferences.loading,
+});
+
+export default connect(mapStateToProps, null)(Main);
```

Comment:
Is there a reason that this didn't get added directly to the component like you did for the `Profile` page?

Reply:
I am not sure about this one. @cgduncan7 can we move this setup to the component file?

Reply:
because there was a unit test that failed and I didn't want to change the test 😬

Reply:
Fair enough 😄